src/App.tsx

```diff
@@ -12,73 +12,74 @@ import Dashboard from "./Modules/DashboardModule/components/Dashboard/Dashboard"
 import ProjectList from "./Modules/ProjectModule/components/ProjectList/ProjectList";
 import TaskList from "./Modules/TasksModule/components/TaskList/TaskList";
 import UsersList from "./Modules/UsersModule/components/UsersList/UsersList";
-import BaceUrlContext from "./Context/BaceUrlContext";
 import ProtectedRoute from "./Modules/SharedModule/components/ProtectedRoute/ProtectedRoute";
 
 function App() {
-  const routes = createBrowserRouter([
-    {
-      path: "/",
-      element: <AuthLayout />,
-      errorElement: <Notfound />,
-      children: [
-        {
-          index: true,
-          element: <Login />,
-        },
-        {
-          path: "login",
-          element: <Login />,
-        },
-        {
-          path: "forget-pass",
-          element: <ForgetPass />,
-        },
-        {
-          path: "reset-pass",
-          element: <ResetPass />,
-        },
-        {
-          path: "register",
-          element: <Register />,
-        },
-        {
-          path: "verify-account",
-          element: <VerifyAccount />,
-        },
-      ],
-    },
-    {
-      path: "dashboard",
-      element: <MasterLayout />,
-      errorElement: <Notfound />,
-      children: [
-        {
-          index: true,
-          element: <Dashboard />,
-        },
-        {
-          path: "projects",
-          element: <ProjectList />,
-        },
-        {
-          path: "tasks",
-          element: <TaskList />,
-        },
-        {
-          path: "users",
-          element: <UsersList />,
-        },
-      ],
-    },
-  ]);
-  return (
-    <>
-         <BaceUrlContext>
-             <RouterProvider router={routes} />
-        </BaceUrlContext>
-    </>
-  );
+	const routes = createBrowserRouter([
+		{
+			path: "/",
+			element: <AuthLayout />,
+			errorElement: <Notfound />,
+			children: [
+				{
+					index: true,
+					element: <Login />,
+				},
+				{
+					path: "login",
+					element: <Login />,
+				},
+				{
+					path: "forget-pass",
```

Comment:
why note `forget-password`?  pass only is confusing because there is a word called pass, but when calling our route `forget-password` this make us strict about what we really mean about our names.